projects/ngrx.io/content/guide/signals/signal-store/private-store-members.md

```diff
@@ -0,0 +1,67 @@
+# Private Store Members
+
+SignalStore allows defining private members that cannot be accessed from outside the store by using the `_` prefix.
+This includes root-level state slices, computed signals, and methods.
+
+<code-tabs linenums="false">
+<code-pane header="counter.store.ts">
+
+import { computed } from '@angular/core';
+import {
+  patchState,
+  signalStore,
+  withComputed,
+  withMethods,
+  withState,
+} from '@ngrx/signals';
+
+export const CounterStore = signalStore(
+  withState({
+    count1: 0,
+    // 👇 private state slice
+    _count2: 0,
+  }),
+  withComputed(({ count1, _count2 }) => ({
+    // 👇 private computed signal
+    _doubleCount1: computed(() => count1() * 2),
```

Comment:
nit: (applies to the test file as well) - the other two examples are a non-private and private example, respectively. However,  the `double` examples are private and then non-private. I think for consistency, In my opinion, I would change these two files to be `doubleCount` and `_doubleCount2` respectively.

Reply:
hey Michael,

This was intentional. I wanted to show that private members can be used to create public ones and vice versa.

Reply:
That makes sense, sounds good. Thank you.